globals.go

```diff
@@ -660,6 +660,7 @@ var (
 		TriggerRequester,
 		GitTagRequester,
 		AdHocRequester,
+		CreateVersionEndpointRequester,
```

Comment:
AdHoc and CreateVersionEndpoint are in both System and User requester type slices. I think it's okay but should they be? In my mind, the AdHoc is system while the other is user, to avoid escalation in the future maybe?